h/browser/chrome/manifest.json

```diff
@@ -28,6 +32,8 @@
   "background": {
     "persistent": true,
     "scripts": [
+      "lib/pdfHandler.js",
+
```

Comment:
I'd kill this whitespace. No need to keep everything so separate. Just put it in line with the rest.